mm2src/coins/lp_coins.rs

```diff
@@ -2843,35 +2844,43 @@ pub enum CoinProtocol {
 
 pub type RpcTransportEventHandlerShared = Arc<dyn RpcTransportEventHandler + Send + Sync + 'static>;
 
-/// Common methods to measure the outgoing requests and incoming responses statistics.
+#[async_trait]
```

Comment:
Can we revert the comment removed?

Reply:
[done](https://github.com/KomodoPlatform/komodo-defi-framework/pull/1966/commits/a193f5df4c896d819068243ef934d0bb8df912c3)